### DISCUSSION TITLE
How do I set vcpkg to use an external MSYS2 for its building ?

### DISCUSSION THREAD

**Discussion by haquocviet** [microsoft/vcpkg#20005] [Q&A]
Currently when the vcpkg builds a library that requires MSYS2, it will download the MSYS2 from online and install. How can I set the vcpkg to use an external pre-installed MSYS2 instead of downloading online?

Thanks.

**Comment by PhoebeHui** [comment]
AFAIK, vcpkg doesn't allow to use pre-installed msys2 on the path, it may cause some unknown issues.
